Modules/Microsoft365DSC/DSCResources/MSFT_SPOApp/MSFT_SPOApp.schema.mof

```diff
@@ -6,7 +6,7 @@ class MSFT_SPOApp : OMI_BaseResource
     [Write, Description("This will deploy/trust an app into the app catalog.")] boolean Publish;
     [Write, Description("Overwrites the existing app package if it already exists.")] boolean Overwrite;
     [Write, Description("Present ensures the site collection exists, absent ensures it is removed"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] string Ensure;
-    [Write, Description("Credentials of the SharePoint Global Admin"), EmbeddedInstance("MSFT_Credential")] string GlobalAdminAccount;
+    [Write, Description("Credentials of the SharePoint Global Admin"), EmbeddedInstance("MSFT_Credential")] string Credential;
```

Comment:
SharePoint admin

Reply:
Replaced across the board by:
Credentials of the account to authenticate with